website/assets/css/homepage.css

```diff
@@ -60,31 +172,47 @@ h1, h2, h3, h4, h5, h6 {
 }
 
 .md-guide-feature {
-  padding-top: 30px;
+  padding-top: 40px;
   padding-bottom: 40px;
-  font-size: 16px;
+  font-size: 15px;
   line-height: 1.5;
   text-align: center;
-  background-color: #fff;
+  background-color: #;
 }
 
 .row {
+  position:relative;
+  max-width:80%;
   margin-top: -15px;
-  margin-right: -15px;
-  margin-left: -15px;
+  margin-right: auto;
+  margin-left: auto;
+
 }
 
 .md-guide-feature p, h1, h2, h3, h4, h5, h6 {
-    color: #000;
+    color: #555;
 }
 
 .icon {
-  font-size: 80px;
+  font-size: 88px;
   color: #03A7DD;
   height: 100px;
   margin-bottom: 20px;
+  margin-right: 20px;
 }
 
+.feature-box .icon {
+  font-size:30px;
+  position:absolute;
+  left:15px;
+  top:15px;
+  width:88px;
+  text-align:center;
+  color:#3b99e0;
+}
+
```

Comment:
may want to clean-up extra whitespace here

Reply:
No, It doesn't need.